app/models/manageiq/providers/ansible_tower/inventory/target/automation_manager.rb

```diff
@@ -1,60 +0,0 @@
-class ManageIQ::Providers::AnsibleTower::Inventory::Target::AutomationManager < ManagerRefresh::Inventory::Target
-  def inventory_groups
-    collections[:inventory_groups] ||= ManagerRefresh::InventoryCollection.new(
-      :model_class    => ManageIQ::Providers::AutomationManager::InventoryRootGroup,
-      :association    => :inventory_root_groups,
-      :parent         => @root,
```

Comment:
really like how `@root` (single refresh specific variable) was removed from the collection definition.

Would be nice if `Persister` definition was reusable. (future thought)